pepper-apis/src/main/java/org/broadinstitute/ddp/constants/ConfigFile.java

```diff
@@ -193,4 +193,10 @@
     public static class Auth0LogEvents {
         public static final String AUTH0_LOG_EVENTS_TOKEN = "auth0LogEvents.token";
     }
+
+    public static class Auth0LogEventApi {
```

Comment:
Looks like an unrelated artifact from another branch. I recommend doing a rebase to `develop`!

Reply:
It doesn't look like latest `develop` branch has these changes, but I think it's fine to merge -- it's only constants. However, if you want, you can play with git:

I typically do a rebase in this situation. Using the `--interactive/-i` flag, I will remove all the other commits except for my own. In this case, I see you have commits `a0b57b4b` and `1a2b1966`.

```
$ git checkout develop
$ git pull
$ git checkout <your_branch>
$ git rebase -i develop
... an editor will pop open in your terminal ...
... remove any line that's not your commits ...
```